spec/google_code_parser_spec.rb

```diff
@@ -21,7 +28,7 @@
 
   describe "#newest_download_version" do
     it "returns the last URL for the platform" do
-      expect(parser.newest_download_version).to eq Gem::Version.new("2.4")
+      expect(parser.newest_download_version).to eq Gem::Version.new("2.42")
     end
 
     context "out-of-order versions" do
```

Comment:
This context is pretty unimportant now that it's just reading from a file, but I left it in for now. Let me know if I should just get rid of it!

Reply:
Agree! I'll delete it.